### PR TITLE
Add a default "userinfo" audience for android docs

### PR DESCRIPTION
**Pull request by lbalmaceda**
This affects android articles and quickstarts.

### CODE REVIEW

File: articles/quickstart/native/android/04-user-profile.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ AuthenticationAPIClient authClient = new AuthenticationAPIClient(auth0);
 It's suggested that you add both the Auth0 `domain` and `clientId` to the `strings.xml` file rather than hardcode them.
 :::
 
-Next, use the `access_token` to obtain the user id with the `AuthenticationAPIClient`. Although the call returns a `UserProfile` instance, this is a basic OIDC conformant profile and the only value guaranteed to be present is the `sub` claim that indicates the user id. We're going to use this value to call later the [Management API](https://auth0.com/docs/api/management/v2#!/Users) and get a full profile.
+Next, use the `access_token` to obtain the user id with the `AuthenticationAPIClient`. Although the call returns a `UserProfile` instance, this is a basic OIDC conformant profile and the only value guaranteed to be present is the `sub` claim that indicates the user id, but depending on the requested scope this may vary. We're going to use this value to call later the [Management API](https://auth0.com/docs/api/management/v2#!/Users) and get a full profile.
```
Comment (cocojoe) [inline articles/quickstart/native/android/04-user-profile.md]: Although the call returns a `UserProfile` instance, this is a basic OIDC conformant profile and the only guaranteed claim is the `sub` which contains the user's id, but depending on the requested scope the claims returned may vary. We're going to use the `sub` value to call the [Management API](https://auth0.com/docs/api/management/v2#!/Users) and return a complete user profile.

File: articles/quickstart/native/android/_includes/_lock_login.md [hunk 2]
```diff
@@ -2,12 +2,16 @@
 
 At this point, you're all set to implement the login in any activity you want. Inside the activity, add the `onCreate`, `onDestroy`, and `callback` methods. Note that we customize the scheme to `demo` as required by the Callback URL defined also in the intent-filter.
 
+Additionally, to make the server return Open ID Connect compliant responses we must either request an `audience` or flip the OIDC Conformant switch on in the client's dashboard Advanced OAuth settings. You can read more about that [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).
```
Comment (cocojoe) [inline articles/quickstart/native/android/_includes/_lock_login.md]: To ensure an Open ID Connect compliant responses you must either request an `audience` or enable the **OIDC Conformant** switch in your Auth0 dashboard under `Client / Settings / Advanced OAuth`. You can read more about this [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).

File: articles/quickstart/native/android/_includes/_login.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 In our login method we create a new `Auth0` instance to hold the credentials. Then by using the `WebAuthProvider` class we can authenticate with any connection enabled for our client in the Auth0 dashboard. We also tell the provider to use the custom scheme `demo` to construct the expected **Callback URL**.
 
+Additionally, to make the server return Open ID Connect compliant responses we must either request an `audience` or flip the OIDC Conformant switch on in the client's dashboard Advanced OAuth settings. You can read more about that [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).
```
Comment (cocojoe) [inline articles/quickstart/native/android/_includes/_login.md]: To ensure an Open ID Connect compliant responses you must either request an `audience` or enable the **OIDC Conformant** switch in your Auth0 dashboard under `Client / Settings / Advanced OAuth`. You can read more about this [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).

File: articles/libraries/auth0-android/index.md [hunk 4]
```diff
@@ -77,13 +77,16 @@ The Authentication Client provides methods to authenticate the user against Auth
 AuthenticationAPIClient authentication = new AuthenticationAPIClient(account);
 ```
 
+Additionally, to make the server return Open ID Connect compliant responses we must either request an `audience` or flip the OIDC Conformant switch on in the client's dashboard Advanced OAuth settings. You can read more about that [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).
+
 ### Login with database connection
 
-Logging in with a database connection merely requires calling `login` with the user's email, password, and the name of the connection you wish to authenticate with. The response will be a Credentials object.
+Logging in with a database connection merely requires calling `login` with the user's email, password, and the name of the connection you wish to authenticate with. The response will be a Credentials object. By requesting the "userinfo" audience we guarantee an Open ID Connect compliant response from the Server.
```
Comment (cocojoe) [inline articles/libraries/auth0-android/index.md]: Logging in with a database connection requires calling `login` with the user's *email*, *password*, and the *connection* you wish to authenticate with. The response will be a Credentials object. By specifying the *audience* an Open ID Connect compliant response will be yielded during authentication.

**Review by cocojoe — CHANGES_REQUESTED** [review]
In general especially in a QS you want to talk about the user as performing the actions themselves, so 'you' instead of 'we'. 

Also try to avoid words like 'basically, merely, simply' before describing an action.  It may be 'simple' but in general you want to avoid implying a level of difficulty.

File: articles/libraries/auth0-android/index.md [hunk 5]
```diff
@@ -77,13 +77,16 @@ The Authentication Client provides methods to authenticate the user against Auth
...
 
 ```java
 authentication
     .login("info@auth0.com", "a secret password", "my-database-connection")
+    .setAudience("https://${account.namespace}/userinfo")
```
Comment (cocojoe) [inline articles/libraries/auth0-android/index.md:89]: You can setAudience for a database connection? Presume this is your p1 or p2 flag?

File: articles/libraries/auth0-android/index.md [hunk 6]
```diff
@@ -77,13 +77,16 @@ The Authentication Client provides methods to authenticate the user against Auth
 AuthenticationAPIClient authentication = new AuthenticationAPIClient(account);
 ```
 
+Additionally, to make the server return Open ID Connect compliant responses we must either request an `audience` or flip the OIDC Conformant switch on in the client's dashboard Advanced OAuth settings. You can read more about that [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).
```
Comment (cocojoe) [inline articles/libraries/auth0-android/index.md]: To ensure an Open ID Connect compliant responses you must either request an `audience` or enable the **OIDC Conformant** switch in your Auth0 dashboard under `Client / Settings / Advanced OAuth`. You can read more about this [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).

File: articles/libraries/lock-android/index.md [hunk 7]
```diff
@@ -103,6 +103,8 @@ Auth0 auth0 = new Auth0("${account.clientId}", "${account.namespace}");
 auth0.setOIDCConformant(true);
 ```
 
+Additionally, to make the server return Open ID Connect compliant responses we must either request an `audience` or flip the OIDC Conformant switch on in the client's dashboard Advanced OAuth settings. You can read more about that [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).
```
Comment (cocojoe) [inline articles/libraries/lock-android/index.md]: To ensure an Open ID Connect compliant responses you must either request an `audience` or enable the **OIDC Conformant** switch in your Auth0 dashboard under `Client / Settings / Advanced OAuth`. You can read more about this [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).

**Comment by lbalmaceda** [comment]
Thank you sir. 🥇

**Review by cocojoe — APPROVED** [review]

File: articles/libraries/auth0-android/index.md (hunk 5 above)
Comment (lbalmaceda) [reply articles/libraries/auth0-android/index.md:89]: This `login` method takes the OIDC flag from the `Auth0` instance (domain/clientId info). Although the audience will be set in any case, the server will only accept it for p2.

**Pull request merged**